conf/retention_test.go

```diff
@@ -8,13 +8,15 @@ import (
 
 func TestParseRetentions(t *testing.T) {
 	cases := []struct {
-		in  string
-		err bool
-		out []Retention
+		in             string
+		acceptableOrig string
+		err            bool
+		out            []Retention
 	}{
 		{
-			in:  "1s:1d:1h:2,1m:8d:4h:2:1234567890,10m:120d:6h:1:true,30m:2y:6h:1:false",
-			err: false,
+			in:             "1s:1d:1h:2,1m:8d:4h:2:1234567890,10m:120d:6h:1:true,30m:2y:6h:1:false",
+			acceptableOrig: "1s:1d:1h:2:true,1m:1w1d:4h:2:1234567890,10m:17w1d:6h:1:true,30m:2y:6h:1:false", // equivalent to 'in'
```

Comment:
the comment says it's equivalent to `in`, but a `true` is expected to be added to the `in`

Reply:
what do you mean? "true" is default, so is equivalent to not specified

Reply:
yeah it's the default, it's just confusing if the comment says they're equivalent but then the exact string is different.